src/__test__/components/data-management/ShareExperimentModal.test.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {
+  screen, render, fireEvent, waitFor,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import fake from '__test__/test-utils/constants';
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
+import mockAPI, {
+} from '__test__/test-utils/mockAPI';
+
+import ShareExperimentModal from 'components/data-management/ShareExperimentModal';
+
+jest.mock('@aws-amplify/auth', () => ({
+  currentAuthenticatedUser: jest.fn().mockImplementation(async () => ({
+    username: 'mockuser',
+    attributes: {
+      email: 'mock@user.name',
+      name: 'Mocked User',
+    },
+  })),
+}));
+
+describe('Share expeirment modal', () => {
+  const onCancel = jest.fn();
+
+  const customAPIResponse = {
+    [`/access/${fake.EXPERIMENT_ID}`]: () => Promise.resolve(new Response(JSON.stringify([{
+      name: 'Bob',
+      email: 'bob@bob.com',
+      role: 'explorer',
+    },
+    {
+      name: 'Mocked User',
+      email: 'mock@user.name',
+      role: 'owner',
+    }]))),
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    enableFetchMocks();
```

Comment:
You only need to do this once in the script, in the root (outside of thsi function) as its impact will be global.